web/docs/operators/lookup.md

```diff
@@ -0,0 +1,83 @@
+# lookup
```

Comment:
This is missing the new frontmatter.

Reply:
Fixed in https://github.com/tenzir/tenzir/pull/3721/commits/9d6e1412952dfe0ac4724fb7199cf4096375179a